tools/windows-node-installer/README.md

```diff
@@ -62,3 +67,26 @@ The IDs of created instance and security group are saved to the `windows-node-in
 The `wni` destroys all resources (instances and security groups) specified in the `windows-node-installer.json` file. 
 Security groups will not be deleted if they are still in-use by other instances.
 
+
+### Azure Platform
+Create Instance:
+We can create instance with using azure create subcommand, Currently all the flags in create subcommand are optional, i.e they don't expect 
+the user to fill in the values. One can also provide already created resources such IP & NIC names via `ipName`, `nicName` towards node creation 
+if you don't want the installer to create one. The other arguments are `image-id` and  `instance-type` which deals with OS and the size settings 
+for the virtual machine. The `image-id` is a URN which is a combination of Publisher:Offer:Sku:Version ex:`RedHat:RHEL:7.4:latest`. Once the instance is 
+created successfully a file will be created under instance name in the `dir`explaining the steps on accessing the instance. For any reason if it 
+couldn't write the data into a file it writes output into the STDOUT. 
+For more info on the details please visit `--help` on azure create subcommand.
+
+Sample Create Command:
+```bash
+./wni azure create --kubeconfig ~/OpenShift/azure/auth/kubeconfig --image-id MicrosoftWindowsServer:WindowsServer:2019-Datacenter:latest 
+--instance-type Standard_B1s --credentials ~/.azure/osServicePrincipal.json --dir ~/windowsnodeinstaller/
+
+```
+Sample Delete Command:
+```bash
+./wni azure destroy --kubeconfig ~/OpenShift/azure/auth/kubeconfig --credentials ~/.azure/osServicePrincipal.json 
+--dir ~/windowsnodeinstaller/
```

Comment:
Please change this from `~/windowsnodeinstaller/` to `./windowsnodeinstaller/`, id rather folders be created in my working directory than my home directory, especially if im working with different clusters

Reply:
Also the documentation doesn't specify that you need to create the directory ahead of time and the command fails if it isn't. Either:
* Document that it needs to be created manually
* Create the directory if it doesn't exist
Personally I'd prefer the second as it gives me a more pleasant user experience, but thats purely my opinion, and you can choose what makes sense to you.

Reply:
I agree, will create a directory in the pwd if it didn't exist.